app/views/admin/rebate_forms/_form.html.haml

```diff
@@ -1,8 +1,15 @@
 = form_for [:admin, @rebate_form] do |f|
-  = f.label :valuation_id
-  = f.text_field :valuation_id
-  = f.label :token
-  = f.text_field :token
+  - show_fields = ['income','dependants', 'full_name', 'lived_here_before_july_2018', 'has_home_business', 'email', 'phone_number', 'email_phone_can_be_used']
+  - @rebate_form.fields.each do |item|
+    - show_fields.each do |hf|
+      - if hf == item.first
+        = f.label item.first
+        = f.text_field(item.first, :value => item.last)
+
+  = f.hidden_field(:updated_by, :value => current_user.email)
+  = f.hidden_field(:updated_at, :value => DateTime.now)
```

Comment:
This also already managed by rails. We don't need it on the form.

Reply:
@Br3nda ahh, interestingly, it doesnt update the date, which is why I put this here.

Reply:
@Br3nda I've removed it